tasks/sealights/sealights-python/0.1/sealights-python.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sealights-python-instrumentation
+  labels:
+    konflux-ci/sealights: "true"
+    konflux-ci/sealights-language: "python"
+spec:
+  description: >-
+    This Tekton task automates the process of configuring your Python application with Sealights for quality analytics and testing.
+    It retrieves the source code from a trusted artifact, installs Python Sealights agent, configures the app for Sealights using
+    vars from your PipelineRun, scans all .py files, reports scan to Sealights, and stores results to be used later on in testing.
+    The task can be triggered by either Pull Request or other events.
+  results:
+    - name: sealights-bsid
+      type: string
+      description: "A unique identifier generated for the current Sealights build session."
+    - name: sealights-build-name
+      type: string
+      description: "A unique build name generated using the commit SHA and current date to prevent conflicts during test reruns."
+    - name: source-artifact
+      description: The Trusted Artifact URI pointing to the artifact with the application source code.
+      type: string
+  params:
+    - name: source-artifact
+      description: The Trusted Artifact URI pointing to the artifact with the application source code.
+      type: string
+    - name: python-version
+      type: string
+      description: "The Python version to use with the 'ubi8/python' image, in the format (e.g., '311')."
+    - name: component
+      type: string
+      description: "The name of the Konflux component associated with the integration tests."
+    - name: scm-provider
+      type: string
+      default: "github"
```

Comment:
Sorry about the noise, but I just noticed this - do both values `github` and `git` (provided in your [Example Pipeline](https://github.com/konflux-ci/tekton-integration-catalog/pull/86/files#diff-4438fb040197221219ed2255b568a72bdac2560f45649a6324432486381b6773R110)) work?

Looks like flags `scm` and `scmprovider` support different values, but I haven't tested it

Reply:
good catch there are different scm flags I had used scm in this task but I should've user scmprovider

https://sealights.atlassian.net/wiki/spaces/SUP/pages/1721106445/Using+Python+agent+-+Command+Reference

I'll update this now and change the example pipelines value. for scmprovider things like Github, Gitlab work for scm just git is accepted.

Reply:
Just changed and tested this

https://console.dev.redhat.com/application-pipeline/workspaces/ascerra/applications/sealights-todo-list/pipelineruns/sealights-todo-list-on-pull-request-ghbjv

Everything works with the scmprovider flag

Reply:
scm support none and git